classes/class_editorplugin.rst

```diff
@@ -162,7 +162,7 @@ Add a custom type, which will appear in the list of nodes or resources. An icon
 
 When given node or resource is selected, the base type will be instanced (ie, "Spatial", "Control", "Resource"), then the script will be loaded and set to this object.
 
-You can use the :ref:`EditorPlugin.handles<class_EditorPlugin_handles>` to check if your custom object is being edited by checking the script or using 'extends' keyword.
```

Comment:
You're co-maintainer of this repo and don't read its README? ;) https://github.com/godotengine/godot-docs/blob/master/README.md#editing-existing-pages

And file header:
>  .. Generated automatically by doc/tools/makerst.py in Godot's source tree.
 .. DO NOT EDIT THIS FILE, but the doc/base/classes.xml source instead.

It doesn't hurt anyway, just think about making the change in classes.xml or it will be overridden next time we sync.